bot/src/commands.js

```diff
@@ -1,19 +1,32 @@
 import logger from './logger'
 import config from './config'
-import {archive, setTags} from './db'
+import {archive, setFrequency, setTags} from './db'
+
+const tagRegexPattern = /:__([a-zA-Z0-9'_+-]+):/g
 
 const commands = [
   {
     pattern: /^(un)?archive\s/,
-    action: async (match, tags, ts) => {
+    action: async (match, message, tags, ts) => {
       const isArchive = !match[1]
       await archive(tags, isArchive, ts)
     },
   },
+  {
+    pattern: /^frequency\s/,
+    action: async (match, message, tags, ts) => {
+      const tagFrequencyPattern = new RegExp(`${tagRegexPattern.source}\\s+(\\d+)`, 'g')
+      const tagsWithFrequency = []
+      let result
+      while ((result = tagFrequencyPattern.exec(message)) !== null) {
```

Comment:
This would be nicer with node 12's [matchAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll), but we don't have node 12 on Now.